AlembicImporter/Packages/Documentation/com.unity.formats.alembic/Public/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changes in Alembic for Unity
+
+## [0.1.3] - 2018-06-08
```

Comment:
No, the public one should only list the packages that were released to the public. I cannot see 0.1.3 on the production server but 0.1.2 is there.

When we finalize the release the public changelog will only contain one new entry: [1.0.0] with all the changes that went in since the last public release.

The process is not perfect, suggestions are welcome!

Reply:
ok I see, thanks! 0.1.3 is not an official or staging release. I just changed the version number to pass the package validation suite